File: HungryCake.API/Models/Feeds/Feed.cs
using System;
using System.Buffers.Text;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using HungryCake.API.Helpers;

namespace HungryCake.API.Models
{
    public class Feed
    {
        [Required]
        public int Id { get; set; }
        public DateTime Created { get; set; }
        public DateTime LastSuccess { get; set; }
        public DateTime LastFail { get; set; }
        public byte[] Icon { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HungryCake-SPA/src/app/_models/feeds/feed.ts
import { User } from '../user';
import { Filter } from '../filter';

export interface Feed {
    id: number;
    created: Date;
    lastSuccess: Date;
    lastFail: Date;
    icon: Uint8Array;
    newIconPath: string;
    isActive: boolean;
    filter: Filter;
    adjustPostsTime: number;
}
